internal/subscription/core.go

```diff
@@ -344,6 +346,57 @@ func createDelayTopics(ctx context.Context, m *Model, topicCore topic.ICore, top
 	return nil
 }
 
+// Get all the subs and rescale all the Retry/Delay/DLQ topics
+func (c *Core) RescaleSubTopics(ctx context.Context, topicModel *topic.Model, partitions int) error {
+	subList, err := c.List(ctx, Prefix+topicModel.ExtractedProjectID)
+	if err != nil {
+		return err
+	}
+	logger.Ctx(ctx).Info("Sublist: ", subList)
```

Comment:
Add better contextual log messages. Sublist does not specify which package, which function and why this message is being printed.